cmd/minikube/cmd/root.go

```diff
@@ -87,6 +92,10 @@ var RootCmd = &cobra.Command{
 				exit.WithError("logdir set failed", err)
 			}
 		}
+
+		if enableUpdateNotification {
+			notify.MaybePrintUpdateTextFromGithub()
```

Comment:
I feel that upgrade notices should only be part of "minikube start". It makes little sense to pester users elsewhere, IMHO. What do you think?

Reply:
No idea, I just reverted the failing commit

But I normally disable the warning, so yes